### PR TITLE
Add FDCAN2 peripheral to G491

### LINKED ISSUE #913: STM32G491 missing FDCAN2 peripheral
It appears to be missing in the svd from ST (i.e. it is present in the G484 svd, but not the G491)

Would submit a PR myself but I can't find any examples on how one would add entire peripheral with a patch rather than the individual registers.

### PR DESCRIPTION
**Pull request by liamkinne**
Resolves #913

### CODE REVIEW

**Review by adamgreig — APPROVED** [review]
thank you!

**Comment by burrbull** [comment]
Python CI fails after this patch. Don't know why:
https://github.com/rust-embedded/svdtools/actions/runs/6796652591/job/18477192193?pr=178

File: devices/stm32g491.yaml [hunk 1]
```diff
@@ -1,5 +1,20 @@
 _svd: ../svd/stm32g491.svd
 
+_derive:
+  FDCAN2:
+    _from: FDCAN
+    baseAddress: 0x40006800
+
+FDCAN2:
+  add:
```
Comment (burrbull) [inline devices/stm32g491.yaml:9]: _add

Comment (liamkinne) [reply devices/stm32g491.yaml:9]: Fixing with #917

**Pull request merged**